definitions/npm/ramda_v0.x.x/flow_v0.39.x-/ramda_v0.x.x.js

```diff
@@ -121,6 +129,7 @@ declare module ramda {
 
   declare var compose: Compose;
   declare var pipe: Pipe;
+  declare var pipeP: PipeP;
```

Comment:
Yeah, that's possible.  Seems like the maintainers would want to maintain a consistent style though.  Oh well, as long as we match the existing style, we should be ok.